cli/cmd/agent_install_ec2ic_test.go

```diff
@@ -0,0 +1,71 @@
+//
+// Author:: Nicholas Schmeller (<nick.schmeller@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/lacework/go-sdk/lwrunner"
+	"github.com/stretchr/testify/assert"
+)
+
+// Requires AWS credentials in the shell environment
+// Lists runners, sends keys, attempts to connect
+// Example command to run:
+// `aws-vault exec default -- go test -run TestAwsEC2ICDescribeInstances`
+// If AWS credentials are already present in the shell environment, only use:
+// `go test -run TestAwsEC2ICDescribeInstances`
+func TestAwsEC2ICDescribeInstances(t *testing.T) {
+	if _, ok := os.LookupEnv("AWS_SECRET_ACCESS_KEY"); !ok {
+		t.Skip("aws credentials not found in environment, skipping test")
+	}
+
+	cli.LogLevel = "DEBUG"
+	agentCmdState.InstallTrustHostKey = true
+	agentCmdState.InstallTagKey = "CaptureTheFlagPlayer"
+	cli.NonInteractive()
+
+	runners, err := awsDescribeInstances()
+	assert.NoError(t, err)
+
+	wg := new(sync.WaitGroup)
```

Comment:
Why parallelize EC2 instance connect installation and not SSH?

Reply:
This is just to make the test run faster -- if the tester's account has a ton of instances, then it would be tedious to iterate over them serially. And we don't have a unit test for SSH. I guess we could add one, but this is basically an integration test, and we already have those in the agent repo.

Reply:
Maybe it would be worthwhile to do this in the actual CLI. I'm not sure how this will work at scale...

Reply:
Oh, sorry, this is just the test. Please file this one away for later then.

Probably a good idea to consider adding some sort of parallelization (think 10000 instances)